changelog/1180.doc.rst

```diff
@@ -0,0 +1 @@
+Adding some clarifying documentation around the type of :attr:`~AuditLogEntry.extra` when the action is :attr:`~AuditLogAction.overwrite_create`.
```

Comment:
```suggestion
Adding some clarifying documentation around the type of :attr:`AuditLogEntry.extra` when the action is :attr:`~AuditLogAction.overwrite_create`.
```
tiny change, just to make it clear in the rendered docs that this is about audit logs